extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -804,95 +827,85 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     return 1;
   };
 
-  private authContacts = async (acctEmail: string) => {
-    const connectToGoogleRecipientLine = this.addedRecipients[this.addedRecipients.length - 1];
-    this.view.S.cached('input_to').val(connectToGoogleRecipientLine.email);
-    this.removeRecipient(connectToGoogleRecipientLine.element);
-    const authRes = await GoogleAuth.newAuthPopup({ acctEmail, scopes: GoogleAuth.defaultScopes('contacts') });
-    if (authRes.result === 'Success') {
-      this.googleContactsSearchEnabled = true;
-      this.canReadEmails = true;
-      this.view.scopes.readContacts = true;
-      this.view.scopes.readOtherContacts = true;
-      await this.searchContacts(this.view.S.cached('input_to'));
-    } else if (authRes.result === 'Denied' || authRes.result === 'Closed') {
-      await Ui.modal.error('FlowCrypt needs this permission to search your contacts on Gmail. Without it, FlowCrypt will keep a separate contact list.');
-    } else {
-      await Ui.modal.error(Lang.general.somethingWentWrongTryAgain(!!this.view.fesUrl));
-    }
-  };
```

Comment:
Similar here - I'm surprised to see it removed. See it in action here https://github.com/FlowCrypt/flowcrypt-browser/pull/4121#issuecomment-974508721 (or not?)

Reply:
if you mean this button "Enable Google Conact Search" -- it is still there.
![image](https://user-images.githubusercontent.com/23335944/154980295-3af488b3-6e9a-4055-b564-db00e12e51d3.png)
It's defined and handled in another piece of code https://github.com/FlowCrypt/flowcrypt-browser/blob/33b617999bc915a68752d6e85df912bd2d969619/extension/chrome/elements/compose-modules/compose-recipients-module.ts#L726